tests/console/yast2_apparmor.pm

```diff
@@ -38,123 +37,98 @@ sub run {
     }
     assert_screen 'yast2_apparmor_enabled';
     # part 1: open profile mode configuration and check toggle/show all profiles
-    send_key 'alt-n';
+    wait_screen_change { send_key 'alt-n' };
     assert_screen 'yast2_apparmor_profile_mode_configuration';
-    send_key 'alt-o';
-    wait_still_screen(1);
+    wait_screen_change { send_key 'alt-o' };
     assert_screen 'yast2_apparmor_profile_mode_configuration_show_all';
-    wait_still_screen(1);
-    send_key 'tab';
-    wait_still_screen(1);
-    send_key 'down';
-    wait_still_screen(1);
-    send_key 'alt-t';
+    wait_screen_change { send_key 'tab' };
+    wait_screen_change { send_key 'down' };
+    wait_screen_change { send_key 'alt-t' };
     assert_screen 'yast2_apparmor_profile_mode_configuration_toggle';
-    wait_still_screen(1);
-    send_key 'alt-b';
-    wait_still_screen(1);
+    wait_screen_change { send_key 'alt-b' };
 
     # close apparmor configuration
-    send_key 'alt-d';
-    wait_still_screen(1);
+    wait_screen_change { send_key 'alt-d' };
     assert_script_run("systemctl show -p ActiveState apparmor.service | grep ActiveState=active");
 
     # part 2: start apparmor configuration again
-    script_run("yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
```

Comment:
That explanation is good for me, but I also mean that you should add that sentence as comment in the code:

```
# Using type_string instead of script_run because of timeout problems with wait_serial: poo#20202
```